server/server.js

```diff
@@ -1,6 +1,6 @@
 //import express module
 const express = require("express");
-
+const dbConnect=require("./model/dbConnect")
```

Comment:
We don't need this database connection. Pull the latest code from master if conflict arises accept incoming one if you have added that code else accept current.